src/TBuffer.cpp

```diff
@@ -3385,18 +3385,19 @@ void TBuffer::appendBuffer(const TBuffer& chunk)
     append(QString(QChar::LineFeed), 0, 1, Qt::black, Qt::black, TChar::None);
 }
 
-int TBuffer::calcWrapPos(int line, int begin, int end)
+int TBuffer::calculateWrapPosition(int lineNumber, int begin, int end)
 {
-    const QString lineBreaks = ",.- \n";
-    if (lineBuffer.size() < line) {
+    const QString lineBreaks = QStringLiteral("- \n");
```

Comment:
Why aren't we grasping the bleeding nettle and getting down to doing this properly by painting the text onto a big enough pixmap and finding the `QTextBoundaryFinder(QTextBoundaryFinder::Line, text)` position that makes the text not exceed the width that `TBuffer::mWrapAt` multiplied by the `QPainter::fontMetrics().averageCharWidth()` for the font in use gives? :slightly_frowning_face:

Counting `QChar`/`TChar`s is not doing us any favours here...

Reply:
What about Chinese which doesn't use spaces to separate ideographs?
https://www.hillslearning.com/2011/06/15/writing-in-chinese-spacing/

Reply:
I just want to fix it for English, don't want to rework the entire font wrapping algorithm right now.

Reply:
Okay then - we will have to come back here in the near future though...